src/pybind/ceph_daemon.py

```diff
@@ -429,3 +483,147 @@ def list(self, ostr: TextIO = sys.stdout) -> None:
                 prio = self._schema[section_name][name].get('priority') or 0
                 table.add_row((section_name, name, nick, prio))
         ostr.write(table.get_string(hrules=HEADER) + '\n')
+
+class DaemonHistogramWatcher(object):
+    """
+    Given a Ceph daemons'' admin socket paths, poll their performance histogram
+    and show the momentary values of chosen counter of interest in either live
+    or batch mode.
+    """
+    def __init__(self) -> None:
+        self.termsize = Termsize()
+
+    def _handle_sigwinch(self, signo, frame) -> None:
+        self.termsize.update()
+
+    def create_histogram(self, sockets, counter, last, batch):
```

Comment:
Why is this a public method? The only caller I'm able to find is `_print_vals_histogram()`.